internal/k3d/k3d.go

```diff
@@ -70,13 +71,16 @@ func (c *client) runCmd(args ...string) (string, error) {
 
 	if err != nil {
 		if c.verbose {
-			fmt.Printf("Failing command:\n  %s %s\nwith output:\n  %s\nand error:\n  %s\n", binaryName, strings.Join(args, " "), out, err)
+			fmt.Printf("Failing command:\n  %s %s\nwith output:\n  %s\nand error:\n  %s\n", binaryName,
+				strings.Join(args, " "), out, err)
 		}
-		return out, errors.Wrapf(err, "Executing '%s %s' failed with output '%s'", binaryName, strings.Join(args, " "), out)
+		return out, errors.Wrapf(err, "Executing '%s %s' failed with output '%s'", binaryName, strings.Join(args, " "),
+			out)
 	}
 
 	if ctx.Err() == context.DeadlineExceeded {
-		return out, fmt.Errorf("Executing '%s %s' command with output '%s' timed out. Try running the command manually or increasing the timeout using the 'timeout' flag", binaryName, strings.Join(args, " "), out)
+		return out, fmt.Errorf("executing '%s %s' command with output '%s' timed out. Try running the command "+
```

Comment:
Why is the word `Executing` changed to `executing` ?

Reply:
Error messages should always start with lower case in Golang. They dont need to but that a common pattern and now the linter also checks for it